api/rtx/src/uvisor_malloc.c

```diff
@@ -18,6 +18,7 @@
 #include <stddef.h>
 #include <stdio.h>
 #include <reent.h>
+#include <string.h>
```

Comment:
Why was `string.h` added? Maybe `uvisor_malloc.c` needs it in order to build, but nothing in this commit seems to warrant its addition.